server/src/main/java/com/datastrato/gravitino/server/GravitinoServer.java

```diff
@@ -94,9 +94,13 @@ protected void configure() {
       register(AccessControlNotAllowedFilter.class);
     }
 
-    HttpServerMetricsSource httpServerMetricsSource =
-        new HttpServerMetricsSource(MetricsSource.GRAVITINO_SERVER_METRIC_NAME, this, server);
     MetricsSystem metricsSystem = GravitinoEnv.getInstance().metricsSystem();
+    HttpServerMetricsSource httpServerMetricsSource =
+        new HttpServerMetricsSource(
+            MetricsSource.GRAVITINO_SERVER_METRIC_NAME,
+            this,
+            server,
+            metricsSystem.getServerConfig());
```

Comment:
`GravitinoServer` has a variable `serverConfig` can be used, don't need to do this.

Reply:
ok